MobileCenterAnalytics/MobileCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -31,19 +29,21 @@ @interface MSAnalyticsTests : XCTestCase<MSAnalyticsDelegate>
 
 @interface MSAnalytics ()
 
-- (void)channel:(id)channel willSendLog:(id<MSLog>)log;
-- (void)channel:(id<MSChannel>)channel didSucceedSendingLog:(id<MSLog>)log;
-- (void)channel:(id<MSChannel>)channel didFailSendingLog:(id<MSLog>)log withError:(NSError *)error;
+- (void)channel:(id)channel willSendLog:(id <MSLog>)log;
+
+- (void)channel:(id <MSChannel>)channel didSucceedSendingLog:(id <MSLog>)log;
+
+- (void)channel:(id <MSChannel>)channel didFailSendingLog:(id <MSLog>)log withError:(NSError *)error;
 
 @end
 
 @interface MSServiceAbstract ()
 
-- (BOOL) isEnabled;
-- (void) setEnabled:(BOOL)enabled;
+- (BOOL)isEnabled;
 
-@end
+- (void)setEnabled:(BOOL)enabled;
```

Comment:
This one is not on you ;) but you can remove the whole MSAnalytics and MSServiceAbstract class extensions from here. You'll probably need to add the `#import "MSServiceAbstractProtected.h"` import.